src/controllers/controller.bubble.js

```diff
@@ -105,14 +95,15 @@ export default class BubbleController extends DatasetController {
 	 */
   getLabelAndValue(index) {
     const meta = this._cachedMeta;
+    const labels = this.chart.data.labels || [];
     const {xScale, yScale} = meta;
     const parsed = this.getParsed(index);
     const x = xScale.getLabelForValue(parsed.x);
     const y = yScale.getLabelForValue(parsed.y);
     const r = parsed._custom;
 
     return {
-      label: meta.label,
+      label: labels[index] || '',
```

Comment:
Could be that I'm missing something, but I'm not sure that is the correct label? `meta.label` is the dataset label while `labels[index]` would equal `x` or `y` in some cases and be empty in others.

Reply:
`meta.label`

<img width="282" alt="Снимок экрана 2022-08-05 в 12 12 55" src="https://user-images.githubusercontent.com/986756/183033368-36c7193b-1d5d-4e5d-8b32-5449615c9d38.png">

`labels[index]`

<img width="247" alt="Снимок экрана 2022-08-05 в 12 12 33" src="https://user-images.githubusercontent.com/986756/183033382-a76c50bc-e7e2-4944-a592-40d0335b48f5.png">

Reply:
This feels like pretty buggy behaviour.
Maby its better to not disable all the default overrides but make them as simillar as possible. 
Like the doughnut/pie tooltip is now a lot better since it tells which dataset its about but with the bubble chart atm you see a title that is not in the graph so it feels verry weird
![image](https://user-images.githubusercontent.com/39033624/183057462-89f25907-58eb-46ac-938a-29726fee1abc.png)